var/spack/repos/builtin/packages/pexsi/package.py

```diff
@@ -62,14 +62,15 @@ def edit(self, spec, prefix):
             ('@STDCXX_LIB', ' '.join(self.compiler.stdcxx_libs))
         ]
 
+        fldflags = ''
         if '@0.9.2' in self.spec:
-            substitutions.append(
-                ('@FLDFLAGS', '-Wl,--allow-multiple-definition')
-            )
-        else:
-            substitutions.append(
-                ('@FLDFLAGS', '')
-            )
+            fldflags += ' -Wl,--allow-multiple-definition'
+
+        if ('^superlu +openmp' in self.spec
+                or '^openblas threads=openmp' in self.spec):
+            fldflags += ' -fopenmp'
```

Comment:
```suggestion
            fldflags += ' ' + self.compiler.openmp_flag
```
Not all compilers use `-fopenmp`